Kickstarter-iOS/Features/Messages/Controller/MessagesViewController.swift

```diff
@@ -161,6 +194,51 @@ internal final class MessagesViewController: UITableViewController, MessageBanne
     self.present(vc, animated: true)
   }
 
+  private func updateTableViewBottomContentInset() {
+    if let userBlockedBannerHostingController = userBlockedBannerHostingController {
+      self.tableView.contentInset
+        .bottom = userBlockedBannerHostingController.view.bounds
+        .height
+    }
+  }
+
+  private func removeUserBlockedBanner() {
```

Comment:
Having the presentation/dismissal code live inside `MessagesViewController` feels like the wrong place to me, since this is code that's largely acting on the UITabBarController, not this view controller. I'd recommend putting this into a quick extension on `UITabBarController` - then you get a clean call from here, like ``self.tabBarController.showBlockedBanner()``.